src/Module/index.jsx

```diff
@@ -196,9 +196,7 @@ export default class Module extends Component
                 cssProps = { {
                     collapsible     : isCollapsible,
                     collapsed       : isCollapsible && isCollapsed,
-                    error           : hasError,
-                    moduleError     : hasModuleError,
```

Comment:
Hmmm but... `hasError` and `hasModuleError` should be separate things with separate appearance.

The wanted behavior is the following:

- `hasError` adds an error icon in the header (our current implementation also requires `errorMessage` to be set too, in order for the icon to be shown)
- `hasModuleError` shows a red stripe on the left-hand side

The two props are independent from each other.

Now, admittedly, this is slightly confusing. But... it’s the current behavior and we need to keep it (for now at least)